README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Are you going to fill this out?

Reply:
Should be updated in c64e407b40169f724ea038614fe77dad27c4b662 to have the useful information